src/index.js

```diff
@@ -1,31 +1,87 @@
+"use strict";
 // 1. Write a function that accepts an array of numbers. Return a new array with all elements doubled
+const numbers = [1, 2, 3];
+const double = (numbers) => {
+  return numbers.map((num) => num * 2);
+};
+console.log(double(numbers));
 // Ex: double([1,2,3]) => [2,4,6]
 
 // 2. Write a function that accepts an array of numbers. Return a new array with only elements that are even number
+const arrNumbers = [1, 2, 3, 4, 5, 6];
+function filterEven(array) {
+  let evenArray = [];
+  array.filter((num) => {
+    if (num % 2 == 0) {
+      evenArray.push(num);
+    }
+  });
+  return evenArray;
```

Comment:
No need to create an array to push result in. Filter always return a new array for you, just make sure that your return that element in the callback
```
const evenArray = array.filter(...)